src/app/pages/system/view-enclosure/enclosure-disks/enclosure-disks.component.html

```diff
@@ -290,13 +296,23 @@ <h1 class="disk-name {{selectedDisk.name}}">{{selectedDisk.name}}</h1>
                 </div>
               </div>
 
-              <div fxFlex="calc(75% - 16px)" class="detail" fxLayout="column" fxLayoutAlign="start start">
+              <div fxFlex="calc(25% - 16px)" class="detail" fxLayout="column" fxLayoutAlign="start start">
                 <div>
                   <div class="label">{{ 'Description' | translate }}</div>
                   <div class="value">{{selectedDisk.description ? selectedDisk.description : 'None'}}</div>
                 </div>
               </div>
 
+              <div fxFlex="calc(50% - 16px)" class="detail" fxLayout="column" fxLayoutAlign="start start">
+                <div>
+                  <div class="label">{{ 'Rotation Rate' | translate }}</div>
+                  <div class="value">
+                    <ng-container *ngIf="selectedDisk && selectedDisk.rotationrate; else flash">{{selectedDisk.rotationrate}}{{'RPM' | translate}}</ng-container>
```

Comment:
Same here. Remove `selectedDisk &&` which passed already the condition on line 173.